src/components/HomePage/ShowMoreButton.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { IconButton } from "@material-ui/core";
+import { KeyboardArrowDownRounded } from "@material-ui/icons";
+
+const ShowMoreButton = ({ className, onClick }) => (
```

Comment:
I see, but what's your opinion on the styles location? Should I still keep them in the homePageStyles file? For me, I think it' s best because they are quite small and having a lot of small styling files clutters a bit, on the other hand, I like the separation of concerns :s

Reply:
On further consideration, I think this makes sense, sort of, to be done like this. If you check the code, what the class is doing, is styling the wrapper div of the button, so that it can be used in more scenarios (you can customize the position, alignment, etc) but not to change the internal look of the button (I mean, it can, if you really want to, but that would be the same as having it in the general .css file and targeting that MUIButton class specifically. Therefore, I think we should keep it this way.

Reply:
I have no problem with several different (but related) components including the same file. I have one, however, with components passing styling concerns to children - imo each component should be concerned with its own styling, at most being able to pass some options (like bootstrap-style colors: `primary`, `secondary`, etc) but other than that not really messing with the look of their children.

Reply:
(The comment section didn't update with the new comment 🙃 )

Wouldn't it make more sense for the parent component to be responsible for wrapping the button and aligning it as necessary then?

Reply:
That's what I was thinking now, I probably makes sense to do it that way instead

Reply:
:shipit: